include/picongpu/fields/MaxwellSolver/ArbitraryOrderFDTD/Weights.hpp

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2013-2020 Klaus Steiniger, Sergei Bastrakov
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <cstdint>
+#include <cmath>
+#include "picongpu/plugins/radiation/utilities.hpp"
+
+
+namespace picongpu
+{
+namespace fields
+{
+namespace maxwellSolver
+{
+namespace aoFDTD
+{
+/** Compute weights of finite differences in
+ *
+ * @tparam T_neighbors Number of neighbors used to calculate
+ *                     the derivative from finite differences.
+ *                     Order of derivative approximation is
+ *                     2 * T_neighbors
+ */
+template< uint32_t T_neighbors >
+struct AOFDTDWeights
+{
+    HDINLINE constexpr AOFDTDWeights()
+    {
+        namespace powSpace = ::picongpu::plugins::radiation::util;
+        // Set initial value
+        weights[0] = 4.0_X * T_neighbors *
```

Comment:
shouldn' the weight be `0<w<1`. If I check this equation I get:
```
( 4 * N * ((2N)!/(2^(2N) (N!)^2 ))^2 ) 

N = 1 --> 1
N = 2 --> 9/8
N = 3 --> 75/64
```

Reply:
According to Ghrist2000 and Vincenti2016 it is correct that the weight of the nearest neighbor is larger than on for solvers of higher order.